README.md

```diff
@@ -37,48 +37,48 @@ This project chooses best practices as defaults:
 | Individual test execution timing                                                          | ✔️       | [Lib](http://hackage.haskell.org/package/hspec-slow)        | [Lib](https://hackage.haskell.org/package/tasty-stats)           |
-| Test suite execution timing                                                               | ✔️       | ✔️                                                           | ✖️                                                                |
-| Helpful output to find slow tests                                                         | ✔️       | [Lib](http://hackage.haskell.org/package/hspec-slow)        | C
+| Test suite execution timing                                                               | ✔️       | ✔️                                                           | ✔️                                                                |
+| Helpful output to find slow tests                                                         | ✔️       | [Lib](http://hackage.haskell.org/package/hspec-slow)        | C                                                                |
 | Coloured output                                                                           | ✔️       | ✔️                                                           | ✔️                                                                |
 | Colourless output                                                                         | ✔️       | ✔️                                                           | ✔️                                                                |
-| `ppShow` instead of `show` for counterexample output                                      | ✔️       | ✖️                                                           | ✖️                                                                |
-| `show` for counterexample output                                                          | 🚧      | ✔️                                                           | ✔️                                                                |
-| Fancy Unicode output                                                                      | ✔️       | ✖️                                                           | ✖️                                                                |
+| `ppShow` instead of `show` for counterexample output                                      | ✔️       | ✖️                                                           | C                                                                |
+| `show` for counterexample output                                                          | 🚧      | ✔️                                                           | [Lib](https://hackage.haskell.org/package/tasty-hunit)           |
```

Comment:
This was a bit difficult because what I want to capture is "when you're using a tasty-based test suite, what do you get?".